includes/LevelAnimation.hpp

```diff
@@ -38,15 +38,7 @@ class LevelAnimation
 
 public:
 
-    /**
-     * @brief pure virtual method for level animation rendering; this method
-     * is called by the game controller
-     *
-     * @param context shared pointer to the current context to use
-     * @param level shared pointer to the level to animate
-     * @param floor the current floor to display in the animation
-     */
-    virtual void render(
+    virtual void renderAnimation(
```

Comment:
rename the renderAnimation() method and render() method, explain the differences between each of them